src/components/proposal/ProposalTableInstrumentScientist.tsx

```diff
@@ -139,55 +135,22 @@ const ProposalTableInstrumentScientist: React.FC = () => {
     {
       title: 'Time allocation',
       render: (rowData) =>
-        `${rowData.technicalReview?.timeAllocation}(${rowData.call?.allocationTimeUnit}s)`,
+        `${rowData.technicalTimeAllocation}(${rowData.allocationTimeUnit}s)`,
       hidden: false,
     },
     {
       title: 'Technical status',
-      render: (rowData) =>
-        rowData.technicalReview
-          ? getTranslation(rowData.technicalReview.status as ResourceId)
-          : '',
+      render: (rowData) => rowData.technicalStatus,
     },
     {
       title: 'Submitted',
       render: (rowData) => (rowData.submitted ? 'Yes' : 'No'),
     },
-    { title: 'Status', field: 'status.name' },
-    {
-      title: 'Deviation',
-      field: 'deviation',
-      hidden: true,
-      render: (rowData: Proposal): number =>
-        standardDeviation(getGrades(rowData.reviews)),
-      customSort: (a: Proposal, b: Proposal) =>
-        (standardDeviation(getGrades(a.reviews)) || 0) -
-        (standardDeviation(getGrades(b.reviews)) || 0),
-    },
-    {
-      title: 'Absolute Difference',
-      field: 'absolute',
-      hidden: true,
-      render: (rowData: Proposal): number =>
-        absoluteDifference(getGrades(rowData.reviews)),
-      customSort: (a: Proposal, b: Proposal) =>
-        (absoluteDifference(getGrades(a.reviews)) || 0) -
-        (absoluteDifference(getGrades(b.reviews)) || 0),
-    },
-    {
-      title: 'Average Score',
-      field: 'average',
-      hidden: true,
-      render: (rowData: Proposal): number =>
-        average(getGrades(rowData.reviews)),
-      customSort: (a: Proposal, b: Proposal) =>
-        (average(getGrades(a.reviews)) || 0) -
-        (average(getGrades(b.reviews)) || 0),
-    },
```

Comment:
Looks like these columns are gone with no replacement. Are we sure we want to do that? Are they a requirement for the ESS?

Reply:
I believe I got permission to remove these from Fredrick on the slack: https://ess-eric.slack.com/archives/C019J2U53U2/p1637655078002200?thread_ts=1637597466.002100&cid=C019J2U53U2